build/download-charts.sh

```diff
@@ -61,6 +61,16 @@ function retrieveIstioRelease() {
   )
 }
 
+function sed() {
+  state=$(git diff)
+  command sed "$@"
+  difference=$(diff <(echo "${state}") <(git diff) || true )
+  if [[ -z "${difference}" ]]; then
+    echo "ERROR: nothing changed, sed seems to not have matched. Exiting"
```

Comment:
Would be nice if this showed the actual sed command that failed (maybe just print out $@).

Reply:
I included the failing call and found some other issues with the git compare- I'm just straight up diffing the file itself now, PTAL